Makefile

```diff
@@ -15,6 +15,12 @@ RC_MUTABLE_IMAGE_NAME  = $(DOCKER_REPO)$(BASE_IMAGE_NAME):canary
 REL_IMAGE_NAME         = $(DOCKER_REPO)$(BASE_IMAGE_NAME):$(REL_VERSION)
 REL_MUTABLE_IMAGE_NAME = $(DOCKER_REPO)$(BASE_IMAGE_NAME):latest
 
+ifeq ($(REL_VERSION),)
+REL_VERSION="devel"
```

Comment:
We should use a new variable for this and not re-use `REL_VERSION`. The `docker-push-release` target (used for pushing official, semantically versioned releases) checks whether this var is set and fails if it isn't. Now that check will always pass and the value that we previously expected to always be a semantic version string could now be "devel" instead.

The approach is fine... just use a separate variable.